package.json

```diff
@@ -39,10 +39,11 @@
     "semver": "^7.5.3",
     "xcode": "3.0.1",
     "xml-js": "^1.6.11",
-    "yargs": "^16.2.0"
+    "yargs": "^17.7.2"
```

Comment:
Yargs 17 dropped support for Node 14 in https://github.com/yargs/yargs/pull/2384 which is why we didn't bump it earlier. I'd say though that bumping it is probably fine, I would just do it in a separate PR and cut a major release for the wizard (no big deal imo). Does that make sense?

Reply:
I had to update it when I was experimenting but it seems we can stay at the current version with what I ended up. So just reverted the version update 👍